package.json

```diff
@@ -1314,14 +1314,14 @@
     "@babel/core": "^7.25.8",
     "@babel/parser": "^7.25.8",
     "@babel/traverse": "^7.25.7",
-    "@mongodb-js/compass-components": "^1.34.2",
-    "@mongodb-js/connection-form": "^1.47.2",
-    "@mongodb-js/connection-info": "^0.11.3",
+    "@mongodb-js/compass-components": "^1.34.5",
+    "@mongodb-js/connection-form": "^1.47.5",
```

Comment:
This has a breaking change so will likely break build (it did end up ignoring my use of `!`), I am going to handle it separately in an upcoming PR in a bit